### PR TITLE
feat: Add to Calendar - iCal

### LINKED ISSUE #5072: Feature: Add to Calendar
Please add the feature 
- [x] "Add to Calendar" to the system
- [x] Position it above the event links on the right hand side of a public event page
- [x] Also include a time in date box as below in the example

- [x] Google Calendar
- [x] iCal
- [ ] Yahoo
- [ ] Outlook

Compare: https://eventyay.com/e/16fa59c7

Screenshot from 2020-09-15 12-04-12

This can be done in a similar fashion as below.

Screenshot from 2020-09-15 12-02-40

### PR DESCRIPTION
**Pull request by akash2408**
Fixes #5072 

#### Short description of what this resolves:
i added yahoo and outlook calendar and dropdown
dropdown
yahoo
outlook

#### Changes proposed in this pull request:

-
-
-

#### Checklist

- [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by mariobehling** [comment]
Please improve as follows:
* add ical which is the most important we need right now
* Outlook is not working, please help to fix it
* Yahoo not working neither on test page

Screenshot from 2020-11-01 22-37-28

**Comment by akash2408** [comment]
i have a doubt in ical calendar.
iCal
can i use this url ? @mariobehling

**Comment by akash2408** [comment]
> i have a doubt in ical calendar.
> iCal
> can i use this url ? @mariobehling @iamareebjamal

**Comment by iamareebjamal** [comment]
There is already an ical URL in the event model

**Comment by akash2408** [comment]
> There is already an ical URL in the event model

I added the iCalUrl from the event model.

File: app/components/public/add-to-calender.hbs [hunk 1]
```diff
@@ -9,7 +9,16 @@
       {{/if}}
       ({{this.timezone}})
       <br>
-      <a href={{this.calendarUrl}} target="_blank" rel="noreferrer nofollow">{{t 'Add to Google Calendar'}}</a>
+      <UiDropdown class="pointing">
+	<a href="#">{{t 'Add to Calendar'}}</a>
+	<div class="menu">
+	  {{#each this.calendarUrls as |calendar|}}
+	    <a href={{calendar.url}} class="item">
```
Comment (iamareebjamal) [inline app/components/public/add-to-calender.hbs:16]: I have no idea why you remove target blank and rel nofollow noreferrer. There is a reason why I added that

**Comment by akash2408** [comment]
Ohh Sorry i just forget that

**Comment by mariobehling** [comment]
When I download the ical it is missing `.ics`. I am getting a file  `1f720482-Open Source Day 2020 - Forum  digitale_.ics`. Please add the ics ending. Thanks!

**Comment by mariobehling** [comment]
I am referring to this example event https://eventyay.com/e/1f720482

**Comment by iamareebjamal** [comment]
I'm getting `1f720482-Open Source Day 2020 - Forum für digitale Souveränität-Calendar.ics`. Which browser were you using?

**Comment by akash2408** [comment]
> When I download the ical it is missing `.ics`. I am getting a file `1f720482-Open Source Day 2020 - Forum digitale_`. Please add the ics ending. Thanks!

ics

**Pull request merged**